src/boot/axios.js

```diff
@@ -5,9 +5,24 @@ const templateLibraryApiClient = axios.create({
   timeout: 15000,
 });
 
+const api = axios.create({
+  baseURL: '/api',
+  timeout: 15000,
+});
+
 templateLibraryApiClient.interceptors.response.use(
   (response) => Promise.resolve(response),
   (error) => Promise.reject(error),
 );
 
-export { axios, templateLibraryApiClient };
+api.interceptors.response.use(
+  (response) => Promise.resolve(response),
```

Comment:
`Promise.resolve(({ data }) => data),`

Reply:
done

Reply:
It's `(response) => Promise.resolve((response.data)),`

Reply:
NO it's `({ data }) => Promise.resolve(data),` =D